frontend/src/Components/Content/Albums.tsx

```diff
@@ -1,39 +1,48 @@
+import React from "react"
 
 export type Album = {
-  name: string,
-  imgUrl: string,
-  description: string
+    name: string,
+    imgUrl: string,
+    description: string
 }
 
-export default function AlbumComponent({ albums }: { albums: Array<Album> }) {
-  function generateInnerContent(album: Album) {
-    return (
-        <>
-            <div style={AlbumNameAlignStyle}>
-                <div style={AlbumNameStyle} translate="no">{album.name}
+type AlbumProps = {
+    albums: Array<Album>
+}
+
+export const AlbumComponent: React.FC<AlbumProps> = ({ albums }) => {
+
+    // this is former code. Leaving in for documentation.
+    // export default function AlbumComponent({ albums }: AlbumProps) {
+    // export default function AlbumComponent({ albums }: { albums: Array<Album> }) {
```

Comment:
Go ahead and delete. This has no utility other than showing us the old way we did it, which we don't want to use anymore.

Reply:
deleted